### PR TITLE
Updates @cothority/byzcoin import

### PR DESCRIPTION
**Pull request by nkcr**
In the beautifier I couldn't use ChainConfig for some strange reasons because it would fail the Calypso tests. This situation is fixed by changing the  ClientTransaction imports.

### CODE REVIEW

**Comment by ineiti** [comment]
Nice catch! Any idea why that happened? I looked at it but couldn't see why this helps...

**Comment by nkcr** [comment]
No idea, this is some kind of sorcery to me 🧙‍♂️.
I got this intuition because there were an error from calypso-rpc.ts saying that `ClientTransaction` were undefined.

**Comment by nkcr** [comment]
Be aware that this change removes the default export of `ClientTransaction`. This change goes toward only using `index` for imports, which is something that already brought small incompatibility in the past when Linus made the change in some parts, without too much trouble (but a lot of frustration..).

**Review by tharvik — CHANGES_REQUESTED** [review]
You actually don't need to import from `[...]/byzcoin/index` but simply from `[...]/byzcoin`, as the `index.ts` is already exporting what's needed.

**Comment by ineiti** [comment]
Waiting for cleanup of `/index` imports

**Review by ineiti — COMMENTED** [review]
Like @tharvik - remove the `/index` from the imports.

File: external/js/cothority/src/byzcoin/beautifier/config.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
+import { Argument } from "../";
+import { ChainConfig } from "../";
```
Comment (ineiti) [inline external/js/cothority/src/byzcoin/beautifier/config.ts:2]:
```suggestion
import { Argument, ChainConfig } from "../";
```

**Review by tharvik — CHANGES_REQUESTED** [review]
Can also you remove the dangling slash at the end of imports path?

**Review by tharvik — APPROVED** [review]

**Pull request merged**